File: stack/stack.py
class Stack:

    ##Stack constructor##
    #Initializes our list for data
    def __init__(self):
        self.stack_list = [];

    ##push method##
    #adds a piece of data to the end of list
    def push(self, data):
        self.stack_list.append(data)

    ##pop method##
    #removes and returns the last element in the stack
    def pop(self):
        return self.stack_list.pop()

    ##peek method##
    #returns the last element in the stack without removal
    def peek(self):
        return self.stack_list[len(self.stack_list) - 1]

    ##isEmpty method##
    #returns true if the stack is empty
    def isEmpty(self):
        return self.stack_list == []

    ##size method##
    #returns the size / length of the stack
    def size(self):
        return len(self.stack_list)

File: stack/stack.js
//Stack constructor
function Stack() {
    this.stack_list = [];
}

Stack.prototype.push = function(data) {
    this.stack_list.push(data);
};

//Will throw an error if stack_list is empty!
Stack.prototype.pop = function() {
    return this.stack_list.pop();
}

//Will throw an error if stack_list is empty!
Stack.prototype.peek = function() {
    return this.stack_list[this.stack_list.length - 1]
}

Stack.prototype.isEmpty = function() {
    return this.stack_list.length === 0;
}

Stack.prototype.size = function() {
    return this.stack_list.length;
}

//TESTING FUNCTIONS

function print(data) {
    console.log(data);
}

print("Initializing Stack");
st = new Stack();
print("Pushing 'World'");
st.push("World");
print( "Peek: " + st.peek() );
print( "isEmpty: " + st.isEmpty() );
print("Pushing 'Hello'");
st.push("Hello");
print( "Size: " + st.size() );
print( "Pop: " + st.pop() );
print( "Pop: " + st.pop() );
print( "isEmpty: " + st.isEmpty() );